internal/distro/rhel85/distro.go

```diff
@@ -529,87 +579,24 @@ func newDistro(name, modulePlatformID, ostreeRef string) distro.Distro {
 		filename: "installer.iso",
 		mimeType: "application/x-iso9660-image",
 		packageSets: map[string]rpmmd.PackageSet{
-			"build": installerBuildPackageSet(),
-			"packages": {
+			buildPkgsKey: x8664InstallerBuildPackageSet(),
+			osPkgsKey: {
 				Include: []string{"lvm2", "policycoreutils", "selinux-policy-targeted"},
 				Exclude: []string{"rng-tools"},
 			},
-			"installer": installerPackageSet(),
+			installerPkgsKey: installerPackageSet(),
 		},
 		rpmOstree: false,
 		bootISO:   true,
 		pipelines: tarInstallerPipelines,
 		exports:   []string{"bootiso"},
 	}
 
-	edgeCommitImgTypeAarch64 := imageType{
-		name:     "edge-commit",
-		filename: "commit.tar",
-		mimeType: "application/x-tar",
-		packageSets: map[string]rpmmd.PackageSet{
-			"build":    edgeBuildPkgSet,
-			"packages": edgeCommitAarch64PkgSet,
-		},
-		enabledServices: edgeServices,
-		rpmOstree:       true,
-		pipelines:       edgeCommitPipelines,
-		exports:         []string{"commit-archive"},
-	}
-	edgeOCIImgTypeAarch64 := imageType{
-		name:     "edge-container",
-		filename: "container.tar",
-		mimeType: "application/x-tar",
-		packageSets: map[string]rpmmd.PackageSet{
-			"build":     edgeBuildPkgSet,
-			"packages":  edgeCommitAarch64PkgSet,
-			"container": {Include: []string{"httpd"}},
-		},
-		enabledServices: edgeServices,
-		rpmOstree:       true,
-		pipelines:       edgeContainerPipelines,
-		exports:         []string{"container"},
-	}
-	edgeInstallerImgTypeAarch64 := imageType{
-		name:     "edge-installer",
-		filename: "installer.iso",
-		mimeType: "application/x-iso9660-image",
-		packageSets: map[string]rpmmd.PackageSet{
-			"build":     edgeBuildPkgSet,
-			"packages":  edgeCommitX86PkgSet,
-			"installer": edgeInstallerPkgSet,
-		},
-		enabledServices: edgeServices,
-		rpmOstree:       true,
-		bootISO:         true,
-		pipelines:       edgeInstallerPipelines,
-		exports:         []string{"bootiso"},
-	}
-	x86_64.addImageTypes(tarImgType, tarInstallerImgTypeX86_64, edgeCommitImgTypeX86_64, edgeInstallerImgTypeX86_64, edgeOCIImgTypeX86_64)
-	aarch64 := architecture{
-		name:   "aarch64",
-		distro: rd,
-		packageSets: map[string]rpmmd.PackageSet{
-			"boot": aarch64BootPackageSet(),
-		},
-	}
-	aarch64.addImageTypes(tarImgType, edgeCommitImgTypeAarch64, edgeOCIImgTypeAarch64, edgeInstallerImgTypeAarch64)
+	x86_64.addImageTypes(qcow2ImgType, vhdImgType, vmdkImgType, openstackImgType, amiImgType, tarImgType, tarInstallerImgTypeX86_64, edgeCommitImgType, edgeInstallerImgType, edgeOCIImgType)
+	aarch64.addImageTypes(qcow2ImgType, openstackImgType, amiImgType, tarImgType, edgeCommitImgType, edgeOCIImgType)
+	ppc64le.addImageTypes(qcow2ImgType, tarImgType)
+	s390x.addImageTypes(qcow2ImgType, tarImgType)
```

Comment:
Wouldn't it make sense to remove the `tar` image type from the `s390x` architecture for now due to https://github.com/osbuild/osbuild-composer/issues/1220 and also to get rid of https://github.com/osbuild/osbuild-composer/pull/1536/files#diff-db2b7786ec8724b03d99b6209d350aab45742d7207450a0122420b17a4cc6e2eR274 ?

Reply:
This shouldn't be an issue in RHEL 8.5 now with the new pipelines.  I've removed the check in the tests instead.